.travis.yml

```diff
@@ -26,6 +26,16 @@ sudo: false
 matrix:
   fast_finish: true
   exclude:
+  - node_js: "5"
```

Comment:
if we're going to add these, we should also add them for node 6, and also remove the iojs ones.